config/prow/cluster/test_pods_pdb.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The purpose of the PodDisruptionBudget here is to never allow evicting pods created by prow.
+# Eviction of pods can happen for one of two reasons:
+# * Cluster autoscaler downscaling
+# * Someome/Something using `kubectl drain`
+#
+# It is still possible to delete the pods via a normal delete call. See https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#the-eviction-api
+
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
```

Comment:
Need to declare the usage of this yaml file, something like:

https://github.com/kubernetes/test-infra/blob/1606c2c0f03778659bdb25e72c390f673b0597a4/config/prow/cluster/BUILD.bazel#L74-L78

(and also one in the service cluster, which is where we run test-infra-trusted jobs)

Reply:
Fixed, PTAL

Reply:
We still need a copy for the service cluster where trusted jobs run.

Reply:
Can you point me to a sample as how to do that? I tried to do it by copying the rule without the `cluster = BUILD_CLUSTER` argument, but that always failed with a rule name clash

Reply:
Looks like the abstraction doesn't currently support this https://github.com/kubernetes/test-infra/blob/e04e876f2a35a7770af7ee0ec500301b1ae830c6/prow/def.bzl#L155-L162

Each thing needs to have a unique name, but right now the name is a function of the file name. You could add a `clusters=` And then do for cluster in clusters: n = cluster+"_"+n or something